fuse.py

```diff
@@ -483,6 +491,11 @@ def _wrapper(func, *args, **kwargs):
             print_exc()
             return -EFAULT
 
+    def _handle_nopath(self, path):
```

Comment:
Yes, I agree that what `path.decode(self.encoding)` does is too trivial to put in a function by itself.

I also think that it's too trivial to influence the name of `_handle_nopath()`.

The decoding isn't what's important, it's cleanly handling the case when no path is given.

If my argument hasn't swayed you and you feel strongly about this then I will change it. I really feel that renaming it to something with "decode" in the name will confuse others when they see that it's not used in all of the operations.